packages/@aws-cdk/aws-apigateway/lib/api-key.ts

```diff
@@ -51,7 +51,7 @@ export interface ApiKeyProps extends ApiKeyOptions {
    * A list of resources this api key is associated with.
    * @default none
    */
-  readonly resources?: RestApi[];
```

Comment:
You can now remove the line `[disable-awslint:...` in the comment above.

Reply:
Done!